### DISCUSSION TITLE
Lighthouse Deck Disabled error

### DISCUSSION THREAD

**Discussion by kmkar31** [bitcraze/discussions#1021] [Q&A]
The lighthouse deck on the crazyflie doesn't seem to receive data from the base stations. The console output reads "Cannot communicate with lighthouse bootloader. FPGA not booted, Lighthouse disabled". I've also tried to cold boot the crazyflie but there is no change. Any fixes?
![LH_Console](https://github.com/bitcraze/discussions/assets/62760535/c7cdc692-9e2e-4220-b14f-4ed25c82eb40)

**Comment by ataffanel** [comment]
That looks like a communication problem between the Crazyflie and the deck. What are the LED status on the deck side? If the boot-loader has booted, both BLUE and RED leds should be lit. If so, I suggest looking at the deck connector to see if it has been damaged somehow.

**Reply by David-Glom** [reply]
Hi,
I have exactly the same problem and the problem is not solved, so is there a solution for the problem? Only the blue LED on the Deck is on.

**Reply by knmcguire** [reply]
See my answer here: https://github.com/orgs/bitcraze/discussions/251#discussioncomment-9328287. Seems like you are dealing with antoher issue so it's best to start a fresh thread.
